service/grpc/topic.go

```diff
@@ -37,16 +37,25 @@ func (s *Server) AddTopicEventHandler(sub *common.Subscription, fn common.TopicE
 	return s.topicRegistrar.AddSubscription(sub, fn)
 }
 
+func (s *Server) AddBulkTopicEventHandler(sub *common.Subscription, fn common.TopicEventHandler, maxMessagesCount, maxAwaitDurationMs int32) error {
```

Comment:
There is no validation of the `maxMessagesCount` or `maxAwaitDurationMs`

Reply:
sorry ?

Reply:
Apologies for the delay over the winter break. The parameters defined are not validated, thus if invalid arguments are passed this would cause an unexpected or unhandled error.

Reply:
Oh they might want to pass those int values itself since go is type strict . and by chance if they pass 0 something then dapr will take care of that as I see in their code ```cfg := contribPubsub.BulkSubscribeConfig{
		MaxMessagesCount:   utils.GetIntValOrDefault(req.BulkSubscribeConfig.MaxMessagesCount, defaultMaxMessagesCount),
		MaxAwaitDurationMs: utils.GetIntValOrDefault(req.BulkSubscribeConfig.MaxAwaitDurationMs, defaultMaxAwaitDurationMs),
	}``` It will set its own default value